code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -683,19 +680,48 @@
 /datum/reagent/consumable/maltodextrin
 	name = "Maltodextrin"
 	description = "A common filler found in processed foods. Foods containing it will leave you feeling full for a much shorter time."
-	color = "#ffffff"
+	color = "#d4e1ee"
 	chem_flags = CHEMICAL_RNG_GENERAL
 	taste_mult = 0.1 // Taste the salt and sugar not the cheap carbs
 	taste_description = "processed goodness"
 	nutriment_factor = 0
 	metabolization_rate = 0.05 * REAGENTS_METABOLISM //Each unit will last 50 ticks
+	overdose_threshold = 5
 
 /datum/reagent/consumable/maltodextrin/on_mob_life(mob/living/carbon/M)
 	M.adjust_nutrition(-0.3) //Each unit will match nutriment 1:1 when completely processed
 	..()
```

Comment:
```suggestion
```

This is not necessary code now that maltodextrin directly has its own nutriment value set to a negative value. 
I probably should have caught that sooner than this PR, but this is a good time to fix it since you're touching up maltodextrin anyway.